test/e2e/util_gatewayapi_test.go

```diff
@@ -348,6 +363,9 @@ func buildGWAPICRDFromName(name string) *apiextensionsv1.CustomResourceDefinitio
 	case "referencegrants":
 		kind = "ReferenceGrant"
 		versions = []map[string]bool{{"v1beta1": true}}
+	case "listenersets":
```

Comment:
Since you are adding a new CRD anyway, can you add the GRPCRoute CRD as well?

Reply:
Done.